lambdas/Dockerfile-test

```diff
@@ -0,0 +1,9 @@
+FROM node:8.10-alpine
```

Comment:
Does lambda require this specific version of node?

Reply:
@apoclyps That's the version they're currently running on - https://github.com/apoclyps/my-dev-space/blob/9e87a0166790fde0ccb648c442e3b60eaf7dcdb2/lambdas/meetupcom/serverless.yml#L7